webapp/src/app.tsx

```diff
@@ -3157,9 +3157,13 @@ export class ProjectView
         if (tc) tc.showHint(true, showFullText);
     }
 
-    getExpandedCardStyle(): any {
+    getExpandedCardStyle(flyoutOnly?: boolean): any {
         let tc = this.refs[ProjectView.tutorialCardId] as tutorial.TutorialCard;
-        return tc ? tc.getExpandedCardStyle('top') : null;
+        let margin = 2;
+        if (flyoutOnly) {
+            margin += 2;
+        }
```

Comment:
does this fix https://github.com/microsoft/pxt-arcade/issues/1402?

Reply:
oh yes, this PR fixes it but not this line--it's https://github.com/microsoft/pxt/pull/6088/files#diff-325ec83a257a791d33bd4484fd67fb24R3316